docs/hw/mounts_cases_adapters.md

```diff
@@ -0,0 +1,245 @@
+<details>
+  <summary>3D-Rendering</summary>
+
+  <script src="https://embed.github.com/view/3d/iRobotEducation/create3_docs/main/docs/hw/data/brackets/C3-RudisLabs-Minimal-RPLIDAR-A1M8-Adapter.stl"></script>
+
+</details>
+
+
+
+* [STL (396.5 kB)](data/brackets/C3-RudisLabs-Minimal-RPLIDAR-A1M8-Adapter.stl)
+* [STEP (1.4 MB)](data/brackets/C3-RudisLabs-Minimal-RPLIDAR-A1M8-Adapter.stp)
+
+#### A1 Large Mount
+<details>
+  <summary>3D-Rendering</summary>
+
+  <script src="https://embed.github.com/view/3d/iRobotEducation/create3_docs/main/docs/hw/data/brackets/C3-RPLidar-A1-Mount.stl"></script>
+
+</details>
+
+
+
+* [STL (1.1 MB)](data/brackets/C3-RPLidar-A1-Mount.stl)
+
+#### A1 USB Mount
+<details>
+  <summary>3D-Rendering</summary>
+
+  <script src="https://embed.github.com/view/3d/iRobotEducation/create3_docs/main/docs/hw/data/brackets/C3-RPLidar-A1-USB-Mount.stl"></script>
+
+</details>
+
+
+
+* [STL (465.4 kB)](data/brackets/C3-RPLidar-A1-USB-Mount.stl)
+
+
+## Compute Boards
+### NavQ+ by NXP
+#### Adapter
+<details>
+  <summary>3D-Rendering</summary>
+
+  <script src="https://embed.github.com/view/3d/iRobotEducation/create3_docs/main/docs/hw/data/brackets/C3-RudisLabs-NavQPlus-Adapter.stl"></script>
+
+</details>
+
+
+
+* [STL (1.2 MB)](data/brackets/C3-RudisLabs-NavQPlus-Adapter.stl)
+* [STEP (3.4 MB)](data/brackets/C3-RudisLabs-NavQPlus-Adapter.stp)
+
+#### Case Top
+<details>
+  <summary>3D-Rendering</summary>
+
+  <script src="https://embed.github.com/view/3d/iRobotEducation/create3_docs/main/docs/hw/data/cases/C3-NavQPlus-Top.stl"></script>
+
+</details>
+
+
+
+* [STL (170.5 kB)](data/cases/C3-NavQPlus-Top.stl)
+* [STEP (413.5 kB)](data/cases/C3-NavQPlus-Top.stp)
+
+#### Case Bottom
+<details>
+  <summary>3D-Rendering</summary>
+
+  <script src="https://embed.github.com/view/3d/iRobotEducation/create3_docs/main/docs/hw/data/cases/C3-NavQPlus-Base.stl"></script>
+
+</details>
+
+
+
+* [STL (150.7 kB)](data/cases/C3-NavQPlus-Base.stl)
+* [STEP (130.2 kB)](data/cases/C3-NavQPlus-Base.stp)
+
+
+### NVIDIA® Jetson Xavier NX™ Developer Kit Mount
+#### Adapter Mount
+<details>
+  <summary>3D-Rendering</summary>
+
+  <script src="https://embed.github.com/view/3d/iRobotEducation/create3_docs/main/docs/hw/data/brackets/data/brackets/C3-JetsonXavierNX-Mount.stl"></script>
+
+</details>
+
+
+
+* [STL (338.4 kB)](data/brackets/C3-JetsonXavierNX-Mount.stl)
+* [3MF (95.9 kB)](data/brackets/C3-JetsonXavierNX-Mount.3mf)
+
+
+### Raspberry Pi®
+#### Small Mount
+<details>
+  <summary>3D-Rendering</summary>
+
+  <script src="https://embed.github.com/view/3d/iRobotEducation/create3_docs/main/docs/hw/data/brackets/C3-RPi-Mount-Small-20211022.stl"></script>
+
+</details>
+
+
+
+* [STL (128kB)](data/brackets/C3-RPi-Mount-Small-20211022.stl)
+
+
+#### Large Mount
+<details>
+  <summary>3D-Rendering</summary>
+
+  <script src="https://embed.github.com/view/3d/iRobotEducation/create3_docs/main/docs/hw/data/brackets/C3-RPi-Mount-20211022.stl"></script>
```

Comment:
This one doesn't render for me, and I'm not sure why. It renders properly for me on the rpi hookup page.